flang-rt/lib/runtime/support.cpp

```diff
@@ -19,6 +19,10 @@ bool RTDEF(IsContiguous)(const Descriptor &descriptor) {
   return descriptor.IsContiguous();
 }
 
+bool RTDEF(IsContiguousUpTo)(const Descriptor &descriptor, char dim) {
```

Comment:
char?

Reply:
It should not exceed the `maxRank`, so I thought `char` would be enough.  Would you prefer `int`/`unsigned`?

Reply:
`int`.  Don't use `char` unless it's a character, and don't use `unsigned` when it just means `int` that I hope is not negative.